composer.json

```diff
@@ -15,21 +15,20 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.3 || ^8.0",
         "ext-pdo": "*",
-        "doctrine/dbal": "^2.9.3",
+        "doctrine/dbal": "^2.11 || ^3.0",
         "phpcr/phpcr": "~2.1.5",
         "phpcr/phpcr-utils": "^1.4.1",
-        "jackalope/jackalope": "dev-phpunit-9@dev"
+        "jackalope/jackalope": "^1.4.1"
     },
     "provide": {
         "jackalope/jackalope-transport": "1.3.0"
     },
     "require-dev": {
         "psr/log": "^1.0",
-        "phpcr/phpcr-api-tests": "dev-allow-php8@dev",
-        "phpunit/phpunit": "^7.5.18 || ^8.0 || ^9.0",
-        "phpunit/dbunit": "^4.0"
+        "phpcr/phpcr-api-tests": "^2.1.22",
+        "phpunit/phpunit": "^7.5.18 || ^8.0 || ^9.0"
```

Comment:
I've just discovered that phpunit 7 is necessary to correctly execute tests with lower dependencies: `Jackalope\TestCase::setUp` method was missing the return type in a previous version.

Reply:
can we go for a newer version of jackalope/jackalope to avoid that? imho there is no need to be very open with the jackalope/jackalope versions we allow.